### DISCUSSION TITLE
关于第三方播放器

### DISCUSSION THREAD

**Discussion by caiyusheng1005** [Hiram-Wong/ZyPlayer#53] [General]
能不能把第三方播放器换成vlc或者其他开源播放器，或者让用户自己选择。现在内置的五个播放器实在太差了。

**Comment by Hiram-Wong** [comment]
> 能不能把第三方播放器换成vlc或者其他开源播放器，或者让用户自己选择。现在内置的五个播放器实在太差了。

已支持调用本地播放器，但是如果源要通过嗅探，则不支持
